servers/keycloak/src/main/docker/helper/exportKeycloakHost.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+export KEYCLOAK_SERVICE_HOST=$(ping -c 1 keycloak | grep -Eo -m 1 '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}';)
```

Comment:
[Optiona][Trivial] Add comment here what this regexp does. It's not clear to me when trying that out.